models.rdf

```diff
@@ -0,0 +1,95 @@
+      <rdfs:comment xml:lang="en">
+        An Administrative Set is a grouping of resources that an administrative unit is ultimately
+        responsible for managing.  The set itself helps to manage the items within it.  An Object
+        or Collection may be contained by only one AdministrativeSet.
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://www.w3.org/ns/ldp#Container"/>
+      <rdfs:isDefinedBy rdf:resource="http://fedora.info/definitions/v4/models#"/>
+    </rdfs:Class>
+
+    <rdfs:Class rdf:about="http://fedora.info/definitions/v4/models#Collection">
+      <rdfs:label xml:lang="en">Collection</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        A Collection is a group of resources.  Collections have descriptive metadata, access metadata,
+        and may links to works and/or collections.
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://fedora.info/definitions/v4/models#Resource"/>
+      <rdfs:isDefinedBy rdf:resource="http://fedora.info/definitions/v4/models#"/>
+    </rdfs:Class>
+
+    <rdfs:Class rdf:about="http://fedora.info/definitions/v4/models#Object">
+      <rdfs:label xml:lang="en">Object</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        An Object is an intellectual entity, sometimes called a "work", "digital object", etc.
+        Objects have descriptive metadata, access metadata, may contain files and other Objects as
+        member "components".
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://fedora.info/definitions/v4/models#Resource"/>
+      <rdfs:isDefinedBy rdf:resource="http://fedora.info/definitions/v4/models#"/>
+    </rdfs:Class>
+   
+    <rdfs:Class rdf:about="http://fedora.info/definitions/v4/models#Resource">
+      <rdfs:label xml:lang="en">Resource</rdfs:label>
+      <rdfs:subClassOf rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:isDefinedBy rdf:resource="http://fedora.info/definitions/v4/models#"/>
+    </rdfs:Class>
+
+    <rdfs:Class rdf:about="http://fedora.info/definitions/v4/models#File">
+      <rdfs:label xml:lang="en">File</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        A File is a sequence of binary data and is described by some accompanying metadata.
+        The metadata typically includes at least basic technical metadata (size, content type,
+        modification date, etc.), but can also include properties related to preservation,
+        digitization process, provenance, etc. Files MUST be contained by exactly one Object.
+      </rdfs:comment>
+      <rdfs:isDefinedBy rdf:resource="http://fedora.info/definitions/v4/models#"/>
+    </rdfs:Class>
+
+    <rdf:Property rdf:about="http://fedora.info/definitions/v4/models#hasFile">
+      <rdfs:label xml:lang="en">has file</rdfs:label>
+      <rdfs:comment xml:lang="en">Links to a File contained by this Object.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://fedora.info/definitions/v4/models#Resource"/>
```

Comment:
I think the domain should be fcdm:Object instead of fcdm:Resource, since fcdm:Collection can't have files associated directly any more.